index.html

```diff
@@ -1,26 +1,62 @@
-    <title>Recyclapedia Home</title>
-  </head>
-  <body>
-      <p>Hello World</p>
-      <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>
-  </body>
-</html>
+<!--
+File: index.html
+Kevin Greene, kevin_greene@student.uml.edu
+Student of Umass Lowell Computer Science,
+in course COMP 4620 GUI Programming II Sec 201
+File created: 3/18/2021
+Description:
+-->
+
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+    <title>Recyclapedia Home</title>
+  </head>
+  <body>
+
+      <div class="container">
+          <div class="row"> <!-- parent row. contains sidebar + rest of page -->
+              <!-- Vertical navbar on side of page -->
+              <div class="col-2">
+                  <ul class="nav flex-column nav-pills">
+                      <li class="nav-item">
+                          <a class="nav-link active" href="#">Active</a>
+                      </li>
+                      <li class="nav-item">
+                          <a class="nav-link" href="#">Link</a>
+                      </li>
+                      <li class="nav-item">
+                          <a class="nav-link" href="#">Link</a>
+                      </li>
+                      <li class="nav-item">
+                          <a class="nav-link disabled" href="#">Disabled</a>
+                      </li>
+                  </ul>
+              </div>
+
+              <div class="col-10"> <!-- all of page besides sidebar -->
+                  <div class="row">
+                      <div class="col-12">
+                          <div class="jumbotron jumbotron-fluid">
+                              <div class="container">
+                                  <h1 class="display-4">Recyclapedia</h1>
+                              </div>
+                          </div>
+                      </div>
+                  </div>
+              </div>
+          </div> <!-- parent row close div -->
+      </div> <!-- container close div -->
+
+      <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+      <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+      <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>

```

Comment:
I also added these links that are required for some Bootstrap functionality that I was missing before